php/blocks/controls/class-control-abstract.php

```diff
@@ -261,18 +261,8 @@ public function render_settings_textarea_array( $setting, $name, $id ) {
 	 * @return void
 	 */
 	public function render_settings_post_type_rest_slug( $setting, $name, $id ) {
-		?>
-		<select name="<?php echo esc_attr( $name ); ?>" id="<?php echo esc_attr( $id ); ?>">
-			<?php
-			$rest_slugs = $this->get_post_type_rest_slugs();
-			foreach ( $rest_slugs as $rest_slug => $post_type_name ) :
-				?>
-				<option value="<?php echo esc_attr( $rest_slug ); ?>" <?php selected( $rest_slug, $setting->get_value() ); ?>>
-					<?php echo esc_html( $post_type_name ); ?>
-				</option>
-			<?php endforeach; ?>
-		</select>
-		<?php
+		$values = $this->get_post_type_rest_slugs();
+		$this->render_select( compact( 'setting', 'name', 'id', 'values' ) );
```

Comment:
I'm happy to run with this if you are, just thought I'd point out that many consider `compact()` bad practice.

Reply:
Yeah, good point. I'll look at this now.

Reply:
https://github.com/getblocklab/block-lab/pull/289/commits/bb7ca6d2ef34babe61e3c7627a0de251ed391ce2 replaces the `compact()` calls with arrays.